typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -418,10 +417,14 @@ class ContextFunder {
         const failure = await this.attemptToFundKey(key, chain);
         failureOccurred ||= failure;
       }
+      return failureOccurred;
     });
 
+    let failureOccurred = false;
     try {
-      await Promise.all(promises);
+      failureOccurred = (await Promise.all(promises)).some(
+        (promiseFailure) => promiseFailure,
+      );
```

Comment:
isnt this inverted?

Reply:
I don't think so, this just checks for any failures in the promises